### PR TITLE
Give PR validation a hook to tell dbuild where to resolve scala from

### PR DESCRIPTION
**Pull request by adriaanm**
Tested (as part of IDE validation) here: https://scala-webapps.epfl.ch/jenkins/view/pr-validators/job/pr-scala-integrate-ide/3880/console

### CODE REVIEW

**Comment by adriaanm** [comment]
¡review by @jsuereth, por favor

File: bin/dbuild.properties [hunk 1]
```diff
@@ -12,6 +12,7 @@
 
 [repositories]
   local
+  pr-scala: ${prRepoUrl-https://oss.sonatype.org/content/repositories/snapshots}
```
Comment (jsuereth) [inline bin/dbuild.properties:15]: Everything is awesome but the name :)

File: sbt-on-2.11.x [hunk 2]
```diff
@@ -35,11 +35,6 @@
         set-version: ${vars.scala-version}
         uri:    "ivy:org.scala-lang#scala-compiler;"${vars.scala-version}
       }, {
-        name:  "scala-actors",
```
Comment (jsuereth) [inline sbt-on-2.11.x:38]: Is this gone rom 2.11?   I guess we're not building our test suites anymore, so we don't need it.   Just curious though for if I re-enable the test suite....

**Comment by jsuereth** [comment]
LGTM!  Thanks man.

File: sbt-on-2.11.x (hunk 2 above)
Comment (adriaanm) [inline sbt-on-2.11.x:38]: Ah, actors are used in the sbt test suite? Just looking for the leanest setup, anticipating moving actors out as a module if we can find the time.

**Comment by adriaanm** [comment]
Thanks for the quick turnaround.

File: sbt-on-2.11.x (hunk 2 above)
Comment (jsuereth) [inline sbt-on-2.11.x:38]: No, but specs or scalatest or scalacheck was using them.  I think that one was scalatest....

**Pull request merged**